Tasks/Chandan/chandan.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+
+<head>
+    <title>Chandan Singh Codes</title>
+    <link rel="stylesheet" href="./style.css">
```

Comment:
Please link chandan.css here instead of style.css

Reply:
Please check I just did a new commit